<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Historique</class>
 <widget class="QDialog" name="Historique">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView_2">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>140</y>
     <width>421</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>701</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>SELECTION PAR:</string>
   </property>
   <widget class="QRadioButton" name="radioId">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Id</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioName">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>97</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioAll">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>97</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ALL</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>621</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHistorique">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>20</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Recherche</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>110</y>
     <width>401</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sortie Stock</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtHistorique">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>401</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Entr√©e Stock</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
